### PR TITLE
Ensure we propagate the trace correctly from test-exporter to querier.

### PR DESCRIPTION
**Pull request by tomwilkie**
Also, log the trace id.

Signed-off-by: Tom Wilkie <tom.wilkie@gmail.com>

### CODE REVIEW

**Comment by tomwilkie** [comment]
Much better:

Screenshot 2019-06-30 at 11 51 27

File: pkg/querier/correctness/runner.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ import (
 	opentracing "github.com/opentracing/opentracing-go"
 	otlog "github.com/opentracing/opentracing-go/log"
 	"github.com/prometheus/client_golang/api"
-	"github.com/prometheus/client_golang/api/prometheus/v1"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
```
Comment (bboreham) [inline pkg/querier/correctness/runner.go]: unrelated change

Comment (tomwilkie) [reply pkg/querier/correctness/runner.go]: sorry! automated editor changes...
